acceptance/indy_acceptance/indy-tests/libraries/common.py

```diff
@@ -0,0 +1,159 @@
+"""
+Created on Nov 13, 2017
+
+@author: khoi.ngo
+
+Containing all functions that is common among test scenarios.
+"""
+
+import asyncio
+import json
+from indy import wallet, pool, ledger
+from .constant import Colors, Constant, Message
+
+
+class Common:
```

Comment:
In Python there is no necessity to use classes as containers of static members as it would be in Java.
In Python a module is a natural container for static data and functions.

Reply:
Updated. Please refer to the latest code.